.github/workflows/build.yml

```diff
@@ -0,0 +1,41 @@
+# TODO: figure out how to trigger workflow from vercel for nx
```

Comment:
I think these comments can be removed, it was related to ignoring other apps and building only changed onex